src/Tournaments/TournamentNav.js

```diff
@@ -0,0 +1,61 @@
+import './TournamentNav.scss';
+import React, { useState } from "react";
+import { NavLink, Link } from 'react-router-dom';
+import { login, logout } from '../User/actions';
+import { connect } from 'react-redux';
+
+
+function TournamentNav({isAuthenticated}) {
+    return (
+        <>
+            <div className="user-info-strip">
```

Comment:
będzie potencjalny refactor tego, bo bardzo podobny pasek jest chyba w User Settings (nie przejmuj się tym, to bardziej notatka na przyszłość)